app/helpers/registrations_helper.rb

```diff
@@ -337,4 +337,11 @@ def link_to_transfer(registration)
 
     "#{base_url}/bo/transfer-registration/#{reg_identifier}"
   end
+
+  def link_to_order_copy_cards(registration)
+    reg_identifier = registration.regIdentifier
+    base_url = Rails.configuration.back_office_url
+
+    "#{base_url}/bo/#{reg_identifier}/order-copy-cards"
```

Comment:
With the change made in #279 `Rails.configuration.back_office_url` will return https://admin.wastecarriersregistration.service.gov.uk/bo so I would expect this to end up as https://admin.wastecarriersregistration.service.gov.uk/bo/bo/CBDU123/order-copy-cards

```suggestion
    "#{base_url}/#{reg_identifier}/order-copy-cards"
```

Reply:
I copied-pasted from https://github.com/DEFRA/waste-carriers-frontend/pull/281/files#diff-2114a3a1a0a7465c9e03785e56da70b2R338, shall we update that too?

Reply:
Ooh, good spot. Yes please.

Guess we need to check if there are any more like this lying around.

Reply:
I think there will be :( maybe we should rollback that refactoring for now?